tests/indexed/test_internals.py

```diff
@@ -474,7 +545,25 @@ def test_cond_tensor_associate(enum_shape, batch_shape, plate_shape, event_shape
             == indices_of(actual_right, event_dim=event_dim)
         )
 
-    assert actual_full.shape == enum_shape + batch_shape + plate_shape + event_shape
+    assert (
+        actual_full.shape
+        == enum_shape + batch_shape + plate_shape + event_shape
+    )
     assert actual_full.shape == actual_left.shape == actual_right.shape
     assert (actual_full == actual_left).all()
     assert (actual_left == actual_right).all()
+
+
+def test_undo_split():
```

Comment:
Can you move this test into a new file `tests/counterfactual/test_handlers_explanation.py` and parametrize it by batch and event shapes similar to other tests here?

Reply:
moved away, will parametrize, now testing combinations with other ops.

Reply:
done.